src/main/webapp/WEB-INF/tags/instructor/results/bySectionPanels.tag

```diff
@@ -1,5 +1,5 @@
 <%@ tag trimDirectiveWhitespaces="true" %>
-<%@ tag description="instructorFeedbackResults - by Giver > Question > Recipient or Recipient > Question > Giver" pageEncoding="UTF-8" %>
+<%@ tag description="instructorFeedbackResults - by Giver &#8594; Question &#8594; Recipient or Recipient &#8594; Question &#8594; Giver" pageEncoding="UTF-8" %>
```

Comment:
Can use the → symbol directly as UTF-8 is supported. Example of a UTF-8 character in the current codebase: https://github.com/TEAMMATES/teammates/blob/1bc5f6589a9003ceb9c2c8f2f3c22d6bdaa0d173/src/main/webapp/WEB-INF/tags/instructor/course/studentTableModal.tag#L13

Reply:
Thanks! I must have missed the other UTF-8 characters and was being (unnecessarily) cautious.